ethereum/core/src/main/java/org/hyperledger/besu/ethereum/storage/keyvalue/KeyValueStoragePrefixedKeyBlockchainStorage.java

```diff
@@ -95,7 +95,7 @@ public Optional<BlockHeader> getBlockHeader(final Hash blockHash) {
   @Override
   public Optional<BlockBody> getBlockBody(final Hash blockHash) {
     return get(BLOCK_BODY_PREFIX, blockHash)
-        .map(bytes -> BlockBody.readFrom(RLP.input(bytes), blockHeaderFunctions));
+        .map(bytes -> BlockBody.readWrappedBodyFrom(RLP.input(bytes), blockHeaderFunctions));
```

Comment:
I'm surprised we're using readWrapped here for an individual block - thought it was lists of blocks? Is it because that's how they are stored?

Reply:
Guess it all comes to how we initially write the bodies to the DB and we do it using the wrapper list.

Reply:
OK, so this is internal to besu I guess. My mental model was wrapped === BlockBodiesMessage but I guess it's not that simple :)